rendercv/cli/commands.py

```diff
@@ -123,6 +123,14 @@ def cli_command_render(
             help="Don't generate the PNG file.",
         ),
     ] = False,
+    watch: Annotated[
```

Comment:
We need to append this option to the `cli_render_arguments` below. Also, we need to add a new field to the `RenderCommandSettings` class in the `data.models.rendercv_settings`.